### PR TITLE
Improve the speed of csv export file generation

### PR DESCRIPTION
**Pull request by harobed**
Improve the speed of csv export file generation

### CODE REVIEW

File: modoboa_admin/management/commands/export_identities.py [hunk 1]
```diff
@@ -28,5 +28,5 @@ def handle(self, *args, **kwargs):
         for u in User.objects.all():
             u.to_csv(csvwriter)
 
-        for a in Alias.objects.all():
```
Comment (tonioo) [inline modoboa_admin/management/commands/export_identities.py]: I think the ".all()" is useless in this case.

Comment (harobed) [inline modoboa_admin/management/commands/export_identities.py]: Fixed

**Comment by harobed** [comment]
Bump

**Comment by tonioo** [comment]
Yes sorry, just back from holydays :)

**Pull request merged**